--- connection.py-old	2015-02-09 14:17:13.140717761 -0800
+++ connection.py	2015-02-09 14:23:12.360714459 -0800
@@ -375,7 +375,7 @@
 
     def generate_url(self, expires_in, method, bucket='', key='', headers=None,
                      query_auth=True, force_http=False, response_headers=None,
-                     expires_in_absolute=False, version_id=None):
+                     expires_in_absolute=False, version_id='some-version-id_vp_'):
         if self._auth_handler.capability[0] == 'hmac-v4-s3':
             # Handle the special sigv4 case
             return self.generate_url_sigv4(expires_in, method, bucket=bucket,
